test/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundServiceSpec.scala

```diff
@@ -215,6 +226,17 @@ class OutboundServiceSpec extends AnyWordSpec with Matchers with GuiceOneAppPerS
       await(underTest.sendMessage(messageRequestFullAddressing))
 
       getXmlDiff(messageCaptor.getValue.soapEnvelope, expectedSoapEnvelope()).build().hasDifferences shouldBe false
+      messageCaptor.getValue.destinationUrl shouldBe "http://example.com:1234/CCN2.Service.Customs.EU.ICS.RiskAnalysisOrchestrationBAS"
+    }
+
+    "resolve destination url when sending the SOAP envelope returned from the security service to the connector" in new Setup {
```

Comment:
This would benefit from an explanation why there's no port in the captured destinationUrl. It took me a few minutes to figure that out. Are we testing that a host with no port is resolved OK?

Reply:
yes, having no port but just host is a valid scenario in the url and hence the test above. I'll update the description to reflect that